app/src/main/res/values-fr/strings.xml

```diff
@@ -65,6 +65,13 @@
     <string name="map">Carte</string>
     <string name="hiking">Randonnée</string>
     <string name="climbing">Escalade</string>
+    <string name="show_completed_activities">Afficher vos activités terminées</string>
+    <string name="filter_options">Filtres</string>
+    <string name="filter_activity_type">Filtrer par type d\'Activité :</string>
+    <string name="filter_difficulty_level">Filtrer par niveau de difficulté :</string>
+    <string name="from">de</string>
+    <string name="to">à</string>
+    <string name="filter_distance_range">Filtrer par distance d\'activité :</string>
     <string name="no_friend_activities">Pas d\'acitivités récentes...  Ajoutes des amis pour en voir plus !</string>
```

Comment:
Could you put "Ajoute" instead of "Ajoutes". I know this wasn't your spelling mistake, but I just saw it.

Reply:
okok i'll handle that